tests/runner.py

```diff
@@ -605,7 +605,7 @@ def get_library(self, name, generated_libs, configure=['sh', './configure'], con
     build_dir = self.get_build_dir()
     output_dir = self.get_dir()
 
-    cache_name = name + ','.join([opt for opt in Building.COMPILER_TEST_OPTS if len(opt) < 10]) + '_' + hashlib.md5(str(Building.COMPILER_TEST_OPTS).encode('utf-8')).hexdigest() + cache_name_extra
+    cache_name = name + ','.join([opt for opt in Building.COMPILER_TEST_OPTS if len(opt) < 7]) + '_' + hashlib.md5(str(Building.COMPILER_TEST_OPTS).encode('utf-8')).hexdigest() + cache_name_extra
```

Comment:
What? Why 1) is this filtered in the first place 2) does reducing it from 10 to 7 help?

Reply:
Looks like its so that users can inject -02, and flags like that to generate different library.

My adding of -Werror got caught up in that and all the library names had Werror in them, which I guess is fine but cosmetically I didn't like it.